File: UserManagement.Web/ClientApp/src/models/address.model.ts
export class AddressModel {
    public id: number;
    public countryId: string;
    public streetNumber: string;
    public streetName: string;
    public city: string;
    public state: string;
    public postalCode: string;
    public countryName: string;

    constructor() {
        this.countryId = '';
    }
}

File: UserManagement.Web/Models/AddressModel.cs
namespace UserManagement.Web.Models
{
    public class AddressModel
    {
        public int? Id { get; set; }
        public int? CountryId { get; set; }
        public string CountryName { get; set; }
        public string StreetNumber { get; set; }
        public string StreetName { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string PostalCode { get; set; }
    }
}
